tests/test_windowing.py

```diff
@@ -13,9 +13,13 @@
 
 from tests import check
 
+from pandas.tseries.frequencies import to_offset
```

Comment:
changes to `test_windowing.py` should be in another PR

Reply:
Something got messed up in a merge of main.  I had to rebase to get these to disappear.